src/scenes/Login/Login.js

```diff
@@ -0,0 +1,70 @@
+import React, { useState } from 'react'
+import './_login.scss'
+import fetch from 'node-fetch'
+import PropTypes from 'prop-types'
+import Home from '../Home/Home'
+const Login = ({ setFoundUser }) => {
+  const [username, setUsername] = useState()
+  const [password, setPassword] = useState()
+  const [error, setError] = useState(null)
+  const [loading, setLoading] = useState(false)
+  const loginUser = async () => {
+    return fetch(`http://localhost:3000/login?username=${username}&password=${password}`, {
+      method: 'GET',
+      headers: {
+        'content-type': 'application/json'
+      }
+    })
+      .then(response => response.json())
+  }
```

Comment:
This function will be re-defined every time the component re-render such as when users input username.
so I suggest 2 options:
1.  Apply this logic inside `handleSubmit`, no need to define a new function
2. Just move this function inside hangleSubmit and rename it to `login`
Your choice

![Screen Shot 2021-06-21 at 3 33 26 PM](https://user-images.githubusercontent.com/14868839/122732710-bfd8e980-d2a6-11eb-9ae2-0fccd88e0380.png)

Reply:
Ok